app/config/Switches.scala

```diff
@@ -70,6 +73,8 @@ class Switches(config: LoginConfig) {
       val source = Source.fromInputStream(result.getObjectContent).mkString
       val statesInS3 = Json.parse(source).as[Map[String, SwitchState]]
 
+      statesInS3.filter(_._2 == On).keys.foreach(notifier.sendStillActiveNotification)
+
```

Comment:
This happens every minute right? That seems obnoxious. Every hour might be OK.

Reply:
Perhaps by something as simple as `DateTime.now().minute() % 0 == 0`?

Reply:
Good idea. Done.